mempool/reactor.go

```diff
@@ -116,6 +116,8 @@ func GetChannelShims(config *cfg.MempoolConfig) map[p2p.ChannelID]*p2p.ChannelDe
 				ID:                  byte(MempoolChannel),
 				Priority:            5,
 				RecvMessageCapacity: batchMsg.Size(),
+
+				MaxSendBytes: 5000,
```

Comment:
are these arbitrary?

Reply:
Not really. But they will be removed entirely assuming we ditch the WDRR approach.

Reply:
add a comment saying this? (nit)

Reply:
Saying what exactly?